profiles/pci/network_drivers/profiles.toml

```diff
@@ -7,6 +7,19 @@ device_ids = "4311 4312 4315 4727 4328 4329 432A 432B 432C 432D 0576 4353 4357 4
 priority = 1
 packages = 'broadcom-wl-dkms'
 post_install = """
+    cat <<EOF >/etc/modprobe.d/01-chwd-net-blacklist.conf
+# Do not load modules on boot.
+blacklist b43
+blacklist b43legacy
+blacklist ssb
+blacklist bcm43xx
+blacklist brcm80211
+blacklist brcmfmac
+blacklist brcmsmac
+blacklist bcma
+EOF
+    mkinitcpio -P
+
     modprobe -r b43
     modprobe -r b43legacy
     modprobe -r ssb
```

Comment:
If modules are in use now, they will not be unloaded, so unloading them during `post_install` is useless

Reply:
So we just remove it, or move unload into pre_install?

Reply:
> So we just remove it, or move unload into pre_install?

Remove it.

Reply:
Ok